challenge_1/c/xoumi/ReverseString.c

```diff
@@ -0,0 +1,27 @@
+#include<stdio.h>
+void main()
```

Comment:
I think it's more canonical to use `int` as the return type for which you return the exit status code. What are your thoughts about switching it to that.